frontend/src/components/ui/ImageGallery.tsx

```diff
@@ -0,0 +1,64 @@
+import Slider from 'react-slick';
+import styles from './ImageGallery.module.css';
+import { useRef, useState } from 'react';
+import { ChevronLeft, ChevronRight } from '@mui/icons-material';
+
+type ProductGalleryProps = {
+  images: string[];
+};
+
+const ImageGallery: React.FC<ProductGalleryProps> = ({ images }: { images: string[] }) => {
+    const sliderRef = useRef<any>(null);
```

Comment:
`any` лучше не использовать вообще. Здесь явно просится следующее:
```suggestion
    const sliderRef = useRef<Slider>(null);
```